common/core/src/endpoint.ts

```diff
@@ -4,7 +4,7 @@
 
 'use strict';
 
-export const apiVersion = '2016-11-14';
+export const apiVersion = '2017-06-30';
```

Comment:
Yup.

---
In reply to: [127058566](https://github.com/Azure/azure-iot-sdk-node/pull/61#discussion_r127058566) [](ancestors = 127058566)